commander/test/utils/reader.spec.ts

```diff
@@ -162,5 +164,50 @@ describe('reader', () => {
 				return expect(result).toEqual(['passphrase', 'password', 'data']);
 			});
 		});
+
+		describe('getFileExtension', () => {
+			it('should return a json file extension', () => {
+				const filePath = './some/path.json';
+				const fileExtension = getFileExtension(filePath);
+
+				return expect(fileExtension).toEqual('.json');
+			});
+
+			it('should throw an error if no file extension is passed', () => {
+				const filePath = './some/path';
+				expect(() => getFileExtension(filePath)).toThrow('Not a JSON file.');
+			});
+		});
+
+		describe('readParamsFile', () => {
+			const filePath = './some/path.json';
+			const fileData = {
+				tokenID: '0000000000000000',
+				amount: 100000000,
+				recipientAddress: 'ab0041a7d3f7b2c290b5b834d46bdc7b7eb85815',
+				data: 'send token',
+			};
+
+			beforeEach(() => {
+				jest.spyOn(fs, 'readFileSync').mockReturnValue(JSON.stringify(fileData));
+			});
+
+			it('should read a json file', () => {
+				const result = readParamsFile(filePath);
+				return expect(JSON.parse(result)).toEqual(fileData);
+			});
+
+			it('should have an amount', () => {
```

Comment:
Optional: IMO this and the following expectation could be merged with the 1st case i.e `should read a json file and have an amount and a recipient address` also since they are not being handled separately (if-else or branch logic) in the source and do not impact the test coverage.

Reply:
Updated!